Library/Koala/Koala.swift

```diff
@@ -328,26 +328,32 @@ public final class Koala {
     device: UIDeviceType = UIDevice.current,
     loggedInUser: User? = nil,
     screen: UIScreenType = UIScreen.main,
-    distinctId: String = (UIDevice.current.identifierForVendor ?? UUID()).uuidString,
-    preferredContentSizeCategory: UIContentSizeCategory = UIApplication.shared.preferredContentSizeCategory) {
+    distinctId: String = (UIDevice.current.identifierForVendor ?? UUID()).uuidString) {
       self.bundle = bundle
       self.client = client
       self.config = config
       self.device = device
       self.loggedInUser = loggedInUser
       self.screen = screen
       self.distinctId = distinctId
-      self.preferredContentSizeCategory = preferredContentSizeCategory
 
-      self.observePreferredContentSizeCategory()
+      self.updateAndObservePreferredContentSizeCategory()
   }
 
-  private func observePreferredContentSizeCategory() {
+  private func updateAndObservePreferredContentSizeCategory() {
+    let update = { self.preferredContentSizeCategory = UIApplication.shared.preferredContentSizeCategory }
```

Comment:
Should we use `[weak self]` here just in case?

Reply:
Hmm yeah good question - I _think_ the observer will capture the `update` closure here but that in turn _shouldn't_ capture `self`.

I'll do a couple tests!

Reply:
Yip that did cause a retain cycle, good catch!